cmake/modules/SwiftConfigureSDK.cmake

```diff
@@ -145,6 +145,8 @@ endfunction()
 #   SWIFT_SDK_${prefix}_LIB_SUBDIR          Library subdir for this SDK
 #   SWIFT_SDK_${prefix}_VERSION_MIN_NAME    Version min name for this SDK
 #   SWIFT_SDK_${prefix}_TRIPLE_NAME         Triple name for this SDK
+#   SWIFT_SDK_${prefix}_OBJECT_FORMAT       The object file format (e.g. MACHO)
```

Comment:
Hah!  I think you put this in the wrong commit 🤷‍♂️

Reply:
No, this was an intentional drive-by addition :) It seemed to be missing...